crates/astria-composer/src/executor/mod.rs

```diff
@@ -94,7 +99,19 @@ pub(crate) use builder::Builder;
 const BUNDLE_DRAINING_DURATION: Duration = Duration::from_secs(16);
 
 type StdError = dyn std::error::Error;
-
+#[derive(Debug, thiserror::Error)]
+pub(crate) enum EnsureChainIdError {
+    // <other error variants from getting genesis, etc>
+    #[error("failed to obtain sequencer chain ID")]
+    FailedToGetChainId,
+    #[error("failed to obtain sequencer genesis")]
+    FailedToGetGenesis,
```

Comment:
Commented below, remove the `FailedtoGetGenesis` variant, keep the `FailedtoGetChainId`